rr_light/reflection_refraction.py

```diff
@@ -54,21 +54,32 @@ def get_vars() -> tuple[float,float,float]:
 
     return alfa, n1, n2
 
+def tot_intern_refl(theta, n1, n2):
+    '''Check if we have total internal reflection.
+
+    :param theta: Angle of incidence
+    :param n1: Index of refraction of the first medium
+    :param n2: Index of refraction of the second medium
+    :return: True or False
+    '''
+    if theta >= 45 and n1 > n2:
+        return True
+    else:
+        return False
```

Comment:
Instead of the if with the two return statements you can directly write 

```
    return theta >= 45 and n1 > n2
```

which is the same